content/blog/developer-experience.md

```diff
@@ -0,0 +1,82 @@
+---
+date: 2024-03-15
+title: 'Serverless Workflows: an Automated Developer Experience'
+---
+Great job on installing the Orchestrator plugin and the SonataFlow operator! But what comes next?
+
+If you aim to understand the full development lifecycle of serverless workflows, from zero to production, then you've come to the right place.
+
+Thanks to the Orchestrator functions and automations, developers can now focus solely on building their applications without being burdened by unnecessary cognitive load. Let's delve into how to effectively manage the end-to-end software development lifecycle of serverless workflows,
+leveraging these built-in capabilities. 
+
+## A Reference Architecture for Automated Deployments of Serverless Workflows
+The reference architecture that we're going to describe consists of the following components:
+* `Orchestrator Helm chart`: the installer of RHDH Orchestrator .
+* `Red Hat Developer Hub (RHDH)`: the Red Hat product for Backstage.
+* `Tekton/Red Hat OpenShift Pipelines`: the Kubernetes-Native CI pipeline to build images and deployment configurations.
+* `ArgoCD/Red Hat OpenShift GitOps`: the CD pipeline to deploy the workflow on the RHDH instance.
+* `Quay.io`: the container registry service to store the software images.
+* `SonataFlow platform`: the SonataFlow implementation of the Serverless Workflow specifications, including a Kubernetes operator and the platform services (data index, jobs service).
+* `SonataFlow`: the custom resource representing the workflow.
+* `GitHub workflow repo`: the source code repository of the workflow.
```

Comment:
Are we bound to github or should we state here just `git repo` or `git config repo`?

Reply:
The Helm chart installs only the github provider, and the software template uses GH specific actions to create the repos, so I would say that for now we are bound to it.

Reply:
ok, we need to see how we can remove GH dependency. For now it is OK